fileutil/checksum_test.go

```
@@ -0,0 +1,113 @@
+		if err := os.Remove(filename); err != nil {
+			t.Error(err)
+		}
+	}()
+
+	checksum := NewChecksum(filename)
+
+	transferred, err := checksum.IsFileTransferred()
+	assert.Nil(err)
+	assert.False(transferred)
+
+	err = checksum.SaveState()
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)